src/magic_folder/magic_folder.py

```diff
@@ -35,6 +36,9 @@
     RELPATH,
     validateSetMembership,
 )
+from allmydata.interfaces import (
+    NoServersError,
```

Comment:
We shouldn't introduce new `allmydata` imports (it's not clear to me this will ever get thrown; see below).